tests/console/yast2_i.pm

```diff
@@ -1,34 +1,99 @@
 use base "console_yasttest";
 use testapi;
 
+sub start_yast {
+    my $env = shift || '';
+
+    script_run("$env /sbin/yast2 sw_single; echo yast2-i-status-\$? > /dev/$serialdev");
+    if (check_screen('workaround-bsc924042', 10)) {
+        send_key 'alt-o';
+        record_soft_failure;
+    }
+    assert_screen 'empty-yast2-sw_single';
+
+    send_key "alt-m", 1; # search mode
+    send_key_until_needlematch 'yast2-sw_search_exact_match', 'down', 20;
+
+    send_key "ret", 1;
+    send_key "alt-p"; # back to search textentry
+}
+
+sub end_yast() {
+    # Upgrade tests and the old distributions eg. SLE11 doesn't show the summary
+    unless ( get_var("YAST_SW_NO_SUMMARY") ) {
+        assert_screen 'yast2-sw_shows_summary', 60;
+        send_key "alt-f";
+    }
+
+    # yast might take a while on sle11 due to suseconfig
+    wait_serial("yast2-i-status-0", 60) || die "yast didn't finish";
+}
+
 sub run() {
     my $self    = shift;
-    my $pkgname = get_var("PACKAGETOINSTALL");
+    my $pkgname = 'mc';
```

Comment:
Because PACKAGETOINSTALL also affects other tests and if it is changed to one without a 'Recommends' this test does not make sense (and fails).
Alternatively we could move that recommends stuff to a new test and leave the old yast_i as it was.